tests/testthat/test-load.r

```diff
@@ -0,0 +1,10 @@
+context("Loading")
+
+test_that("Package root is working directory when loading", {
+  expect_message(load_all("testLoadDir"), "[|].*/testLoadDir[|]")
+})
+
+test_that("Loading a package subdirectory throws an error", {
```

Comment:
Hmmm, I think the goal was to make it easy to bootstrap an existing directory into a package - i.e. you could just drag all your files into an R directory and start using devtools immediately.